### ISSUE TITLE
No custom play button in mobile.

### ISSUE THREAD

**Issue by ImranBugCMG** [videojs/videojs-youtube#558]
So the custom play button was missing in mobile and I tracked it down to this piece of code in youtube.js

    poster: function() {
        // You can't start programmaticlly a video with a mobile
        // through the iframe so we hide the poster and the play button (with CSS)
        if (_isOnMobile) {
            return null;
    }

Removing `return null;` and the css fixed the issue for me but wanted to leave this info here since I think this is a bug.

**Comment by gkatsev** [comment]
And it works fine? Previously, on the devices that `isOnMobile` is set to true for you couldn't actually use a custom button.
Also, which device are you testing on?

**Comment by ImranBugCMG** [comment]
@gkatsev Yep works fine, I tested on my iPhone XR. Apart from this I also tested on multiple mobile devices on browserstack.

**Comment by gkatsev** [comment]
Interesting, any particular setup? Like playsinlin or somesuch?

**Comment by csabaxyz** [comment]
I had the same issue. The play button didn't show, and the poster was actually above the video, so there was no way of starting it. The above code snippet fixed it for me as well.

I use the nuevo skin, if that helps: https://www.nuevodevel.com/nuevo/skins
